cypress/support/commands.ts

```diff
@@ -14,3 +15,28 @@ Cypress.Commands.add('injectAxe', () => {
     window.axe = axe;
   });
 });
+
+// Add better logging to cy.tab
+Cypress.Commands.overwrite('tab', (originalFn, subject) => {
```

Comment:
This change isn't strictly necessary, but I wanted better logging of cy.tab. I opened an issue: https://github.com/Bkucera/cypress-plugin-tab/issues/12